process/process.lua

```diff
@@ -367,7 +370,11 @@ function process.handle(msg, _)
       function (msg)
         return msg.Tags.Type == "Process" and Owner == msg.From 
       end,
-      require('.boot')(ao)
+      function ()
+        require('.boot')(ao)
+        -- load all the initial state once
+        require(".state.init").init()
```

Comment:
confirm that this executes on process creation

Reply:
add `print` to state init function

Reply:
https://github.com/ar-io/aos/pull/2/commits/024b223732e5264aaa837f56573a6e71e72fad93